config/locales/views/home/home.yml

```diff
@@ -5,6 +5,9 @@ en:
     more_case_studies: More case studies
     title: Case Studies
   home:
+    audits:
```

Comment:
The original wireframes had this section as being about audits, but the new content is just about Energy Sparks in general rather than audits so might want to rename

Reply:
This is a fair point, thanks